packages/veto-app/frontend/src/layouts/MainLayout/MainLayout.tsx

```diff
@@ -27,7 +27,10 @@ const getActiveMenu = (currentPathName: MainLayoutRoute | string): MainLayoutRou
 }
 
 const MainLayout: FunctionComponent<Props> = ({ path }: Props): JSX.Element => {
-  const [activeMenu] = useState<MainLayoutRoute>(getActiveMenu(window.location.pathname))
+  const [activeMenu, setActiveMenu] = useState<MainLayoutRoute>(getActiveMenu(window.location.pathname))
+  const updateActiveMenu: VoidFunction = (): void => setActiveMenu(getActiveMenu(window.location.pathname))
```

Comment:
Non-blocking: No need to type to `VoidFunction`, the type will be inferred by TS based on the variable